src/stepFunctions/commands/executeStateMachine.ts

```diff
@@ -42,11 +42,12 @@ export async function executeStateMachine(params: {
 
         const baseTemplateFn = _.template(BaseTemplates.SIMPLE_HTML)
         const executeTemplate = _.template(StepFunctionsTemplates.EXECUTE_TEMPLATE)
-        const loadScripts = ExtensionUtilities.getScriptsForHtml(['executeStateMachine.js'])
-        const loadLibs = ExtensionUtilities.getLibrariesForHtml(['vue.min.js'])
-        const loadStylesheets = ExtensionUtilities.getCssForHtml(['executeStateMachine.css'])
+        const loadScripts = ExtensionUtilities.getScriptsForHtml(['executeStateMachine.js'], view.webview)
+        const loadLibs = ExtensionUtilities.getLibrariesForHtml(['vue.min.js'], view.webview)
+        const loadStylesheets = ExtensionUtilities.getCssForHtml(['executeStateMachine.css'], view.webview)
```

Comment:
Form does not appear to render correctly

![image](https://user-images.githubusercontent.com/39839589/83182202-d9772400-a0da-11ea-9841-22192adadacb.png)